internal/bcf/bcf_test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bcf
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestGetReferenceId(t *testing.T) {
+	testCases := []struct {
+		file   string
+		name   string
+		id     int
+		errMsg string
```

Comment:
Testing error messages is generally a bad idea.
If you care about the error, your clients probably will too.  In that case, we should have an error type and a function like IsNotFoundError(err) that does the appropriate type assertion (or you can export a simple value if the error is not dynamic - as in this case)

Reply:
I don't care about the error message, just checking that it returned an error should be enough.